README.md

```diff
@@ -2,7 +2,7 @@
 # What does Splunk Connect for Docker do?
 Splunk Connect for Docker is a plug-in that extends and expands Docker's logging capabilities so that customers can push their Docker and container logs to their Splunk on-premise or cloud deployment.
 
-Splunk Connect for Docker is a supported open source product. Customers with an active Splunk support contract receive Splunk Extension support under the Splunk Support Policy. Splunk Connect for Docker is not supported on windows. Please refer to Docker Engine managed plugin system documentation on support for Microsoft Windows and other platforms as well as Prerequisites in this document. 
+Splunk Connect for Docker is a supported open source product. Customers with an active Splunk support contract receive Splunk Extension support under the Splunk support policy. Splunk Connect for Docker is not supported on Windows. Refer to Docker Engine managed plugin system documentation on support for Microsoft Windows and other platforms and to Prerequisites in this document for more information. 
```

Comment:
> "Splunk Support Policy"  - i think this is a use of a capitalization because a document is a proper noun. 

> and to Prerequisites

 - doesn't flow